conformance_tests/sysman/test_sysman_pci/src/test_sysman_pci.cpp

```diff
@@ -171,9 +171,21 @@ TEST_F(PCI_TEST, GivenSysmanHandleWhenRetrievingPCIStatsThenStatsAreReturned) {
     lzt::destroy_command_queue(cq);
     lzt::destroy_command_list(command_list);
 
-    // delay to prevent identical counters in rare cases
-    std::this_thread::sleep_for(std::chrono::seconds(1));
-    auto pci_stats_later = lzt::get_pci_stats(device);
+    zes_pci_stats_t pci_stats_later{};
+    uint32_t wait = 0;
+    do {
+      pci_stats_later = lzt::get_pci_stats(device);
```

Comment:
If the PCI stats are not available by the time the first iteration occurs, wouldn't the EXPECT_EQ assertion in get_pci_stats() fail?

Reply:
No, it wouldn't fail. The last updated stats will be returned. Hence looping until the timestamp gets updated (This makes sure the stats are refreshed)